pkg/daemons/config/types.go

```diff
@@ -258,10 +258,13 @@ func (a ArgString) String() string {
 	return b.String()
 }
 
-func GetArgsList(argsMap map[string]string, extraArgs []string) []string {
+// GetArgs appends extra arguments to existing arguments overriding any default options.
+func GetArgs(argsMap map[string]string, extraArgs []string) []string {
+	const hyphens = "--"
```

Comment:
nit: we don't generally use consts inside functions in this codebase; move this up into the top of this file.

Reply:
If it's only used in this function, it's convention to keep it within the scope it's used.